scripts/r.in.wms/r.in.wms.html

```diff
@@ -70,8 +70,8 @@ <h4>Open Street Map</h4>
 </pre></div>
 <p>
 More OSM WMS servers can be found online, e.g. on the OSM wiki in a
-<a href="http://wiki.openstreetmap.org/wiki/WMS#Public_WMS_Servers">OSM WMS Servers list</a>
-and on the <a href="http://wiki.openstreetmap.org/wiki/OSM-WMS-EUROPE">OSM-WMS-EUROPE</a> page.
+<a href="https://wiki.openstreetmap.org/wiki/WMS#Public_WMS_Servers">OSM WMS Servers list</a>
+and on the <a href="https://wiki.openstreetmap.org/wiki/OSM-WMS-EUROPE">OSM-WMS-EUROPE</a> page.
```

Comment:
This redirects to https://wiki.openstreetmap.org/wiki/WORLD_OSM_WMS, as the former europe one now has world coverage

Reply:
I suggest to update the various WMS URLs in a separate PR as it needs further work beyond the intention of this PR.

Reply:
I'm fine with that decision too, but the link used now is a wiki redirect to the one I showed. I don't know what extra work is needed, but both are acceptable

Reply:
Ok, updated in 697695f6cf26eef7289aadc54e6bf36cc6cad809